drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/main/java/org/drools/workbench/services/verifier/plugin/client/builders/UtilsTypeResolver.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.services.verifier.plugin.client.builders;
+
+import java.util.Optional;
+
+import org.drools.workbench.models.guided.dtable.shared.model.ConditionCol52;
+import org.drools.workbench.services.verifier.plugin.client.Logger;
+
+public class UtilsTypeResolver
+        implements TypeResolver {
+
+    private final int columnIndex;
+    private final ConditionCol52 conditionColumn;
+    private VerifierColumnUtilities utils;
+
+    public UtilsTypeResolver(final VerifierColumnUtilities utils,
+                             final int columnIndex,
+                             final ConditionCol52 conditionColumn) {
+        this.utils = utils;
+        this.columnIndex = columnIndex;
+        this.conditionColumn = conditionColumn;
+    }
+
+    public String getType(final Optional<String> operatorFromCell)
+            throws ValueResolveException {
+        try {
+            Logger.add("Getting type from utils");
+
+            if (operatorFromCell.isPresent()) {
+                return conditionColumn.getFieldType();
+            }
+
+            final String type = utils.getType(conditionColumn,
+                                              columnIndex);
+
+            Logger.add("Resolver type from utils: " + type);
```

Comment:
Do we want to keep the logging in this class. Asking because we do not log what type was resolved if operator was in the cell.

Reply:
This should not be needed.